docs/sources/installation/fedora.md

```diff
@@ -42,6 +48,12 @@ To update the `docker` package:
 
 Please continue with the [Starting the Docker daemon](#starting-the-docker-daemon).
 
+### Uninstall Docker
+
+To uninstall the `docker` package:
```

Comment:
The `docker` package is in back-ticks here, which (I think) hints at it being the literal name of the software that's removed. However, the package here is called `docker-io`, so this might become confusing.

Perhaps we should just use "Docker package" in all documents, to be consistent and not refer to the  actual name of the package.

/cc @docker/docs-owners